require("pbx") fails

After installing `pbx` and fixing #1, doing `require("pbx")` fails with `Error: Cannot find module 'processor'`.

This is probably due to [src/index.coffee#L1](https://github.com/pandastrike/pbx/blob/master/src/index.coffee#L1) requiring `"processor"` instead of `"./processor"`.

--- a/src/index.coffee
+++ b/src/index.coffee
@@ -1,1 +1,1 @@
-module.exports = require "processor"
+module.exports = require "./processor"

